fedora_messaging/tests/fixtures/callback.py

```diff
@@ -0,0 +1,2 @@
+def rand(message):
+    return 4
```

Comment:
Hahahahaha https://www.xkcd.com/221/ :rofl:

Reply:
missing a docstring with the source :laughing: